src/core/validation/CustomValidationRules.ts

```diff
@@ -47,9 +47,22 @@ export class CustomValidationRules {
             validate: (value) => {
                 const isValidAddress = AddressValidator.validate(value);
                 const isValidAlias = AliasValidator.validate(value);
-                if (isValidAddress || isValidAlias) {
+                if (isValidAddress) {
                     return true;
                 }
+                if (isValidAlias) {
+                    const nameSpaceRepository = AppStore.getters['network/repositoryFactory'].createNamespaceRepository();
```

Comment:
Would it be better to move into the store (Namespace store perhaps), so it can be shared by other components?

Reply:
BTW, https://github.com/nemgrouplimited/symbol-desktop-wallet/pull/589 I have added the linkedAddress in namespace store as needed for the decrypt message